filebeat/module/elasticsearch/gc/ingest/pipeline.json

```diff
@@ -0,0 +1,37 @@
+{
+  "description": "Pipeline for parsing Elasticsearch JVM garbage collection logs",
+  "processors": [
+    {
+      "grok": {
+        "field": "message",
+        "patterns": [
+          "%{TIMESTAMP_ISO8601:timestamp}",
+          "%{GREEDYMULTILINE:elasticsearch.gc.message}"
```

Comment:
I would suggest to put this under just `message`. It means we will overwrite what is in message at the moment. If we want to keep it, we could use `log.message`.

Reply:
I think the missing `elasticsearch.gc.message` part in the expected log file is the reason CI is failing.